src/components/details/market/MarketHistoryEntry.tsx

```diff
@@ -11,6 +12,8 @@ interface Props {
 function MarketHistoryEntry({ activity }: Props) {
   const [from, setFrom] = useState<string>();
   const [to, setTo] = useState<string>();
+  const [unmodifiedFrom, setunmodifiedFrom] = useState<string>();
+  const [unmodifiedTo, setunmodifiedTo] = useState<string>();
```

Comment:
Small nitpick: the "set" function should use full camelCase (`setUnmodifiedX`)

Reply:
Good catch, fixed!